### DISCUSSION TITLE
Strange behavior of Extruder! E Calibration Issues

### DISCUSSION THREAD

**Discussion by meier2009** [mriscoc/Ender3V2S1#690] [General]
I try to Cal my Extruder!

Have dual Gear on Bowden.

Steps 138.6
Results : 105mm to 97mm
not change the Steps!

if i change from 138.6 to maybe 138.4 than the results are from are sometimes 108mm other times 97mm

someone idea if this is hardware or software related ? maybe is miscalculation if there´s a change!

Setup Ender 3 V2 with Mrisoc 2.1.3

dont wanna change now, before figure out what´s the matter!

If i do 5 test in a row, i don´t get the same result!

Mostly relative near together, but than an "extrem" except.

and the strangest thing, if i change the steps only 0.1 its extrem different extrude than before.
what i don´t mentioned, it looks like if i go over 138, that it behave like exponetiell adding something.

if i´m under 138 than the changes are smaler, but also incoherent.

and if i go back to old values, it´s also different.

also, as asked, hardware seem´s okay. new extruder gears, looks everything fine, no viewable faults.

thx

**Comment by dfinberg** [comment]
Any chance there's a worn spot on your extruder gear, so it isn't feeding consistently?

**Reply by meier2009** [reply]
hi, thx for reply!

No, its new and totaly fine!
If i do 5 test in a row, i don´t get the same result!

Mostly relative near together, but than an "extrem" except.

and the strangest thing, if i change the steps only 0.1 its extrem different extrude!
what i don´t mentioned, it looks like if i go over 138, that it behave like exponetiell adding something.

if i´m under 138 than the changes are smaler, but also incoherent.

**Comment by brew99** [comment]
Where are you doing your measurement from?  Remove the bowden tube and measure right after the extruder to eliminate any external issues.   If that doesn't work, you must be getting slip somewhere (tension not enough?).

**Reply by meier2009** [reply]
im doing my measurment at the top of the extruder, like in all hints and tips mentioned :-)
if i remove the bowden the results a way more different than when i extrude slowly. What in my opinion, is the "real" extrude.

maybe its wrong... :-/

**Reply by brew99** [reply]
So are you pushing it through the hotend and melting the filament, but doing the measurement at the entrance to the extruder? I've always removed the bowden tube, so as not to introduce other possible issues. Give it a try and see if it stays repeatable.

You are trying to set the motor steps only, not necessarily compensate for other resistances (of which there should be none to very little). You need to make that measurement repeatable. I run the same firmware, and have no issues with my e-steps (I'm not using a dual gear extruder though)

**Reply by dfinberg** [reply]
right, you want to find esteps, which is a physical characteristic of your extruder motor, so removing the bowden tube simplifies that. Then you can tune flow, which will be affected by your choice of filament after that to calibrate how much is moving out the nozzle.

**Reply by meier2009** [reply]
okay, thx for the hints.
i will try this and also maybe try the new firmware to excluse failure in old version.
